lib/zipTree.js

```diff
@@ -32,6 +32,36 @@ function addTree(zip, src) {
     .then(() => zip); // Original zip for chaining.
 }
 
+/**
+ * Add a directory recursively to a zip file. Files in src will be added to the top folder of zip.
+ * @param {JSZip} zip a zip object in the folder you want to add files to.
+ * @param {string} src the source folder.
+ * @param {Object} noDeploy
+ * @return {Promise} a promise offering the original JSZip object.
+ */
+function addTreeNoDeploy(zip, src, noDeploy) {
```

Comment:
I think we don't need a new function just for that as most of the logic is copied over from the original `addTree`. I think it would be better to just have the optional argument of `noDeploy` to the original function. What do you think?

Reply:
Thanks for the review :)
You are right. I'll fix it along with the test.

Reply:
@pgrzesik  I'm done adding test and fixing the code. Please check it out.